iconservice/icon_service_engine.py

```diff
@@ -1109,13 +1117,15 @@ def _handle_get_iiss_info(self, context: 'IconScoreContext', params: dict) -> di
         response['variable']['rrep'] = reward_rate.reward_prep
 
         calc_start_block, calc_end_block = context.storage.meta.get_last_calc_info(context)
-        if calc_end_block < 0 or context.block.height != calc_end_block:
+
+        if calc_start_block < 0 or context.block.height != calc_end_block:
             calc_end_block: Optional[int] = context.storage.iiss.get_end_block_height_of_calc(context)
```

Comment:
Sorry! this is my mistake. when I supplemented 'get_iiss_info' method, had not considered 'calc_end_block' value. 'calc_end_block' is used at '_create_rc_result' method, so do not allocate other value to it.

Reply:
I see, I'll apply it

Reply:
I see, I'll apply it.

Reply:
I have fixed this bug